app/celer-dump-data.cc

```diff
@@ -614,6 +614,71 @@ void print_atomic_relaxation_data(
     cout << endl;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Print optical material properties map.
+ */
+void print_optical_material_data(ImportData::ImportOpticalMap const& iom)
+{
+    if (iom.empty())
+    {
+        CELER_LOG(info) << "Optical material data not available";
+        return;
+    }
+
+    CELER_LOG(info) << "Loaded optical material data map with size "
+                    << iom.size();
+
+#define POM_STREAM_SCALAR(ID, NAME)                                          \
+    "| " << setw(11) << ID << " | " << setw(24) << #NAME << " | " << setw(7) \
+         << ""                                                               \
+         << " | " << setprecision(3) << setw(9) << NAME << " | " << setw(52) \
+         << ""                                                               \
+         << " |\n"
+#define POM_STREAM_VECTOR(ID, NAME)                                          \
+    "| " << setw(11) << ID << " | " << setw(24) << #NAME << " | " << setw(7) \
+         << NAME.x.size() << " | " << setw(9) << ""                          \
+         << " | (" << setprecision(3) << setw(10) << NAME.x.front() << ", "  \
+         << setprecision(3) << setw(10) << NAME.y.front() << ") -> ("        \
+         << setprecision(3) << setw(10) << NAME.x.back() << ", "             \
+         << setprecision(3) << setw(10) << NAME.y.back() << ") |\n";
+    cout << R"gfm(
+# Material optical property data
+
+| Material ID | Property                 | Size    | Scalar    | Vector endpoints (MeV, value)                        |
+| ----------- | ------------------------ | ------- | --------- | ---------------------------------------------------- |
+)gfm";
+
+    for (auto const& [key, val] : iom)
+    {
+        auto const& scint = val.scintillation;
+        cout << POM_STREAM_SCALAR(key, scint.yield);
+        cout << POM_STREAM_SCALAR(key, scint.resolution_scale);
+        for (auto const& scint_comp : scint.components)
+        {
+            cout << POM_STREAM_SCALAR(key, scint_comp.yield);
```

Comment:
Use the same key [name] for all components?

Reply:
Not sure what you mean? Currently they all have the same `key` (material ID) and the names are the same for each component (i.e., there is no `1`, `2`, `3` suffix added).

Reply:
I meant the same name - can you add the suffix easily or is it intentionally dropped?

Reply:
I can add it.